core/src/main/java/nl/nn/adapterframework/jdbc/JdbcTransactionalStorage.java

```diff
@@ -914,7 +913,7 @@ public void setDateFieldType(String string) {
 		dateFieldType = string;
 	}
 
-	@IbisDoc({"10", "The type of the columns messageId and correlationId, slotId and comments are stored in. N.B. (100) is appended for id's, (1000) is appended for comments.", ""})
+	@IbisDoc({"10", "The type of the columns messageId and correlationId, slotId and comments are stored in. N.B. <code>(100)</code> is appended for id's, <code>(1000)</code> is appended for comments.", ""})
```

Comment:
ik snap de laatste zin niet helemaal, en ik zou `(<code>100</code>)` doen zodat de haakjes niet de code tag krijgen.

Reply:
Maar de haakjes zijn onderdeel van wat toegevoegd wordt.
Er wordt mee bedoeld dat 'char' verandert in 'char(100)'